docs/developer-notes.md

```diff
@@ -1,46 +1,66 @@
-# Getting started
+# Setting up 
+This section guides you through the steps required to set up your computer for developing CATcher.
+
+## Prerequisites
+You need Node 12 to develop this project. 
+Run `node -v` in your OS terminal to check the version of Node on your computer. 
+
+If you do not have Node 12 installed in your computer, click on [this link](https://nodejs.org/en/blog/release/v12.20.0/) to download Node 12. 
+
+## Getting Started
 1. Fork this repository into your Github account.
 
 2. Clone the forked repository into your computer.
 
-3. Make sure you are using Node 12 for this project.
-
-4. Install dependencies with npm: `npm install` 
+3. Install dependencies with npm: Run `npm install`.
 
-5. Compile and start the application: `npm start`
+4. Compile and start the application: Run `npm start`.
 
 # Commands
-Before building the application using the build commands below, go to `index.html` and comment out the necessary `base href` as explained in the file. 
+This section shows you different commands you can run to build the application in different operating systems.
+<div markdown="block" class="alert alert-info">
+
+**:information_source: Before building the application using the build commands below, go to `index.html` and comment out the necessary `base href` as explained in the file. **<br>
```

Comment:
Did you intend this sentence to be in bold?

Reply:
I intended it to be bolded, but I think it is fine if it is not bolded as well! Please let me know which is better and I will change it accordingly.

Reply:
Bold is fine; I was just clarifying since the bold formatting was not rendered correctly